Source/Selling/Mixins/Frame.lua

```diff
@@ -0,0 +1,60 @@
+AuctionatorSellingFrameMixin = CreateFromMixins(AuctionatorCommoditySellingMixin, AuctionatorItemSellingMixin)
+
+local AUCTION_DURATIONS = {
+  [12] = {
+    Duration = 1,
+    Text = AUCTION_DURATION_ONE
+  },
+  [24] = {
+    Duration = 2,
+    Text = AUCTION_DURATION_TWO
+  },
+  [48] = {
+    Duration = 3,
+    Text = AUCTION_DURATION_THREE
+  }
+}
+
+function AuctionatorSellingFrameMixin:OnLoad()
+  Auctionator.Debug.Message("AuctionatorSellingFrameMixin:OnLoad()")
+
+  AuctionatorCommoditySellingMixin.Initialize(self)
+  AuctionatorItemSellingMixin.Initialize(self)
+end
+
+function AuctionatorSellingFrameMixin:OnShow()
+  Auctionator.Debug.Message("AuctionatorSellingFrameMixin:OnShow()")
+end
+
+function AuctionatorSellingFrameMixin:SetDuration(dropdown, duration)
+  ToggleDropDownMenu(1, nil, dropdown.DropDown)
+  UIDropDownMenu_SetText(dropdown.DropDown, AUCTION_DURATIONS[duration].Text)
+  dropdown.DropDown:SetDuration(AUCTION_DURATIONS[duration].Duration)
+  ToggleDropDownMenu(1, nil, dropdown.DropDown)
+end
```

Comment:
There's quite a bit happening in this function. Probably needs a comment or two to explain what this method is doing (I know it's `SetDuration`, but how?)

Reply:
This would just be re-documenting the Blizzard API...

Reply:
np